src/services/distributions.ts

```diff
@@ -2,15 +2,18 @@ import * as Joi from '@hapi/joi';
 import Pino from 'pino';
 import {QueryResult, QueryTermsService} from './query';
 import {Catalog} from '@netwerk-digitaal-erfgoed/network-of-terms-catalog';
+import {IActorInitSparqlArgs} from '@comunica/actor-init-sparql/lib/ActorInitSparql-browser';
```

Comment:
To be sure: is this - `IActorInitSparqlArgs` - the correct type? File `src/services/query.ts` uses `ActorInitSparql`.

Reply:
“Depend on abstractions, not on concretions”, so I used the interface rather than the class here:

```
export declare class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
```

I do agree this interface name is quite confusing, though.

Reply:
👍🏻